### ISSUE TITLE
Fix translation error

### ISSUE THREAD

**Issue by Tmp341** [notepad-plus-plus/notepad-plus-plus#9857]
### Description of the Issue

I've made a small mistake with my translation #9843. I didn't want to create a new PR just for 1 line.

### Expected Behavior

These 2 lines had to be same (if i get it correctly):
 `<Item id="6352" name="Match case"/>`
`<Item id="6332" name="Match case"/>`

### Actual Behavior
I didn't get the exact meaning for id 6352 at the time i posted my PR. If those 2 were meaning same thing (match lower/upper case), we can use 

https://github.com/notepad-plus-plus/notepad-plus-plus/blob/70d7b8504c4a41f96dc0ab0025e77bf4ac460a07/PowerEditor/installer/nativeLang/turkish.xml#L931

for this line too (i've also checked other translations, they were using same words):

https://github.com/notepad-plus-plus/notepad-plus-plus/blob/70d7b8504c4a41f96dc0ab0025e77bf4ac460a07/PowerEditor/installer/nativeLang/turkish.xml#L926

### TLDR;

TLDR; Use **line 931** *(ID 6332)* for **Line 926** *(ID 6352)* too.
